modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/trust/storage/TrustStorage.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.dag.l1.domain.trust.storage
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust._
+
+trait TrustStorage[F[_]] {
+  def getTrustScore(peerId: PeerId): F[Option[Score]]
+  def getTrustRating(peerId: PeerId): F[Option[Rating]]
+  def getTrustObservationAdjustment(peerId: PeerId): F[Option[ObservationAdjustment]]
```

Comment:
when I think about this, I believe the return type can be with no Option type.
if I want to get, let's say` trustScore for peer A, it should fallback to the default value if nothing is stored. so the return type would be `F[Score]` anyway. what do you think @cngo-github?

Reply:
`None != 0`. These `TrustValue`s are meant to be calculated locally, to compliment EigenTrust's global trust values. So, `None` indicates that there is no knowledge of an entity while `0` indicates there is some knowledge, but the value is `0` (i.e., no opinion). So, no knowledge != no opinion.